edgelet/edgelet-docker/src/module.rs

```diff
@@ -116,6 +188,13 @@ impl<C: 'static + Connect> Module for DockerModule<C> {
     }
 }
 
+pub trait DockerModuleTop {
```

Comment:
The most logical place for `top` code is inside `DockerModule` implementation. But as you mentioned I need to mock real code execution in tests.
It's not  a another method inside `Module` trait because it make sense only for docker runtime at least for now.

Reply:
Agree.